### PR TITLE
Fix markers never disappear in Rviz | Add documentation

### PR DESCRIPTION
**Pull request by DavidB-CMU**
When viewing detected tags in Rviz, the marker remains visible forever, even when it is no longer detected. This PR fixes that and adds some documentation.

### CODE REVIEW

File: src/apriltags.cpp [hunk 1]
```diff
@@ -133,7 +133,8 @@ void ImageCallback(const sensor_msgs::ImageConstPtr& msg)
     cv_bridge::CvImagePtr subscribed_ptr;
     try
     {
-        subscribed_ptr = cv_bridge::toCvCopy(msg, "mono8");
+        //subscribed_ptr = cv_bridge::toCvCopy(msg, "mono8");
```
Comment (psigen) [inline src/apriltags.cpp]: Is there a reason to leave this commented here?

File: launch/kinect2.launch [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<launch>        
+  <!-- 
+       Start an apriltag node using the default settings
+       to connect to an OpenNI device (e.g. Kinect). 
+  -->
+  <node pkg="apriltags" type="apriltags" name="apriltags">
+    <param name="~default_tag_size" value="0.037" />
+
+    <param name="~viewer" value="0" />
+    <param name="~publish_detections_image" value="0" />
```
Comment (psigen) [inline launch/kinect2.launch]: Can we just use `bool` types here?
http://wiki.ros.org/roslaunch/XML/param

File: include/apriltags.h [hunk 3]
```diff
@@ -30,7 +32,9 @@ std::string tag_family_name_;
 
 // Settings and local information
 int viewer_;
+int publish_detections_image_;
```
Comment (psigen) [inline include/apriltags.h]: Let's just use a bool.

**Comment by psigen** [comment]
Thanks @DavidB-CMU, this looks great, especially the updated docs.

I left a few comments, only minor cleanups.

**Comment by DavidB-CMU** [comment]
Thanks for the comments @pras 

Regarding
> //subscribed_ptr = cv_bridge::toCvCopy(msg, "mono8"); 
> subscribed_ptr = cv_bridge::toCvCopy(msg, "bgr8"); 

this was a note-to-self, to start a discussion: 
Originally the node is copying the image msg from the camera to a greyscale image. Maybe this was to try to save processing time later, or maybe it was so the debug window showed a mono image with bright white tag overlaid.  
But looking over the entire section of code, I wonder if we need to do any of this image copying at all?

**Comment by DavidB-CMU** [comment]
I've fixed all those things. I'll come back to the image-copying thing later if we are concerned about performance.

**Comment by psigen** [comment]
> subscribed_ptr = cv_bridge::toCvCopy(msg, "bgr8");

@DavidB-CMU I think the purpose of this isn't to make a copy, but because this function will also _convert_ the image type to a mono image.  AprilTags is only using greyscale detection anyway, so this was probably done to ensure that the incoming image was already in a format that would definitely work with the AprilTags library before passing it into the detection call (as opposed to some exotic format that might cause issues).

**Comment by DavidB-CMU** [comment]
That makes sense, I think this PR is good now @psigen

**Pull request merged**